modin/core/io/text/text_file_dispatcher.py

```diff
@@ -49,7 +48,6 @@ class TextFileDispatcher(FileDispatcher):
     read_callback = None
 
     @classmethod
-    @logger_decorator("PANDAS-API", "TextFileDispatcher.get_path_or_buffer", "INFO")
     def get_path_or_buffer(cls, filepath_or_buffer):
```

Comment:
Why do we no longer have the decorator here - are we not logging this anymore, or are we logging this at a different level?

Reply:
FileDispatcher class is being logged, and with the decorator setup we automatically log methods in the child classes by inheritance.